server/lib/host-metrics.js

```diff
@@ -192,16 +186,16 @@ GROUP BY t1."hostCurrency"`,
 
 export async function getHostFeesTimeSeries(host, { startDate, endDate, timeUnit } = {}) {
   const newResults = await sequelize.query(
-    `SELECT SUM(t1."amountInHostCurrency") as "_amount", t1."hostCurrency" as "_currency", DATE_TRUNC('${timeUnit}', t1."createdAt") as "date"
+    `SELECT SUM(t1."amountInHostCurrency") as "_amount", t1."hostCurrency" as "_currency", DATE_TRUNC(:timeUnit, t1."createdAt") as "date"
```

Comment:
@Betree : Sorry I missed providing this review, but in this function we name variables as `t1` but there's no `t2` and such. Maybe it's better to just name the variable to `t` instead? Same applies to `getHostFeeShare` and `getHostFeeShareTimeSeries` I think. 😉  

Except for that everything else looks good to me. 😄

Reply:
I think that part hasn't been touched for some time, I'm not sure why they were named that way. We could indeed rename.